### PR TITLE
Update storage.md

### PR DESCRIPTION
**Pull request by curquiza**
According to v0.21.0 and the current movies.json (that was changed a bit compared to the last time the page was done).
Plus, the page said only 6.3MB was used in RAM -> that was not true.

### CODE REVIEW

**Review by guimachiavelli — DISMISSED** [review]
looks good to me. Thanks for the PR! 😄 

EDIT: on second thought, I prefer to get @dichotommy's approval before merging on the v0.21 branch

File: reference/under_the_hood/storage.md
Comment (dichotommy) [inline reference/under_the_hood/storage.md:59]: Would it not be clearer to say `RAM` instead of `Resident memory size`? I know they are not *exactly* the same, but I believe they are functionally the same for almost all purposes.

```suggestion
| Raw JSON | MeiliSearch database size on disk | RAM use | Virtual memory use |
```

**Review by dichotommy — APPROVED** [review]
Thanks for updating this :)

File: reference/under_the_hood/storage.md
Comment (dichotommy) [reply reference/under_the_hood/storage.md:59]: I suggest changing `size` to `use` to make a distinction between memory and space. Even though that distinction really doesn't exist in the case of virtual memory, I think it's still helpful for the user :)

Comment (curquiza) [reply reference/under_the_hood/storage.md:59]: @Kerollmops, WDYT?

Comment (Kerollmops) [reply reference/under_the_hood/storage.md:59]: It seems ok to me, @dichotommy you can find [more information on this StackOverflow message](https://askubuntu.com/a/112407).

**Pull request merged**
